datadog/fwprovider/resource_datadog_integration_gcp.go

```diff
@@ -299,6 +303,9 @@ func (r *integrationGcpResource) updateState(ctx context.Context, state *integra
 	if privateKeyId, ok := resp.GetPrivateKeyIdOk(); ok {
 		state.PrivateKeyId = types.StringValue(*privateKeyId)
 	}
+	if runFilters, ok := resp.GetCloudRunRevisionFiltersOk(); ok && len(*runFilters) > 0 {
+		state.CloudRunRevisionFilters, _ = types.SetValueFrom(ctx, types.StringType, *runFilters)
```

Comment:
Why ignore the `err` here? Not familiar with the terraform library, but seems unsafe?

Reply:
Spoke offline. The 2nd return param is not `err` but is `diag.Diagnostics`, which I _think_ can represent an error. However, looking all over this repo, it seems the norm is to ignore the 2nd return param. Will look into more later but good for now.